src/App.tsx

```diff
@@ -1,24 +1,251 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { changeTodo, createTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './TodoList';
+import { Status } from './types/Status';
+import { TodosFilter } from './TodosFilter';
+import { CompletedChangeTodo } from './types/CompletedChangeTodo';
 
-const USER_ID = 0;
+const USER_ID = 11623;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(Status.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [value, setValue] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [changedTodos, setChangedTodos] = useState<number[]>([]);
+  const [editFlag, setEditFlag] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState<boolean>(false);
```
suggest more conventional naming for boolean variables